internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -177,6 +178,12 @@ EnvironmentManagerRole:
           ]
           Resource:
             - !Sub 'arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:parameter/copilot/${AppName}/${EnvironmentName}/secrets/*'
+        - Sid: SSMSession
+          Effect: Allow
+          Action:
+            - ssm:StartSession
+          Resource:
+            - !Sub "arn:${AWS::Partition}:ssm:${AWS::Region}::document/AWS-StartPortForwardingSessionToRemoteHost"
```

Comment:
Why do we need both this statement and the other `ssm:StartSession` on `"*"` at L92?

Reply:
`ssm:StartSession` requires permissions to two resources: the SSM document that we use (`arn:${AWS::Partition}:ssm:${AWS::Region}::document/AWS-StartPortForwardingSessionToRemoteHost`) and the other is the ECS task that we target (L92).

Reply:
oops missed the `Condition` block of the other one! SGTM!